lmdeploy/messages.py

```diff
@@ -101,6 +101,14 @@ def special_word_token_ids(words):
             stop_words=special_word_token_ids(gen_config.stop_words),
             bad_words=special_word_token_ids(gen_config.bad_words))
 
+    def __post_init__(self):
```

Comment:
Let's make server return error message to users not crash

Reply:
This will not crash server

Reply:
> This will not crash server

Hi @AllentDan  Do we catch `AssertionError` and handle it when validation fails?

Reply:
Yes, we catch the error. Now, for the api_server, we check request before `__post_init__`